GiEnJul/ClientApp/src/routes/registerGiver/ContactInfo.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
 import { Button, Grid, Container } from '@material-ui/core';
 import useStyles from './Styles';
-import validator from 'validator';
```

Comment:
Have you checked that all of these changes wont interfere with the signup process and that it still connects to the API?